src/test/scala/com/fulcrumgenomics/commons/io/IoTest.scala

```diff
@@ -62,6 +75,15 @@ class IoTest extends UnitSpec {
     Io.assertReadable(Io.StdIn)
   }
 
+  it should "not throw an exception for Unix fifos" in { if (hasMkfifo) {
+    val lines       = Seq("foo", "bar")
+    val pipePath    = Paths.get("test1").toAbsolutePath
```

Comment:
Can you use either `makeTempFile` or something else that will create files in a temp directory, not in the directory the test is running in please?